skyportal/handlers/api/classification.py

```diff
@@ -324,6 +324,7 @@ def get(self, obj_id):
         classifications = obj.get_classifications_readable_by(self.current_user)
         for classification in classifications:
             del classification.groups
+            classification.redshift = obj.redshift
```

Comment:
I think this should be optional, controlled by a query parameter, such as:
```
- in: query
  name: includeRedshift
  nullable: true
  schema:
    type: boolean
    description: |
      Number of candidates to return per paginated request. Defaults to 25
```
Then `include_redshift = self.get_query_argument("includeRedshift", False)` and check on it before adding to the classification object.